### PR TITLE
add pre-publish steps to  Publishing.md

### PR DESCRIPTION
**Pull request by marcdumais-work**
Signed-off-by: Marc Dumais <marc.dumais@ericsson.com>

As suggested by @akosyakov

### CODE REVIEW

File: doc/Publishing.md [hunk 1]
```diff
@@ -2,10 +2,19 @@
...
 - publish packages
 - tag the published version
 
+## Pre-Publishing Steps
+Before publishing it's important to make sure that theia-apps builds against 'next'. Else we will have problems with "latest" after publishing
+
+- Make sure that there is no pending build on Theia master, otherwise a new "next" version might be published while we validate the current "next".
+
+- Go in the theia-apps repo [here](https://github.com/theia-ide/theia-apps/commits/master) and identify the latest commit. There should be an icon next to it; either a red X or a green checkmark. Click on it to go the the travis page. There re-trigger the build. We need to make sure that at least the various "next" builds passes. If it doesn't, it needs to be fixed before continuying.
+
```
Comment (epatpol) [inline doc/Publishing.md]: Typo `continuying` -> `continuing` ?

File: doc/Publishing.md [hunk 2]
```diff
@@ -2,10 +2,19 @@
 
 In order to release one should:
 
+- pre-publishing steps
```
Comment (hexa00) [inline doc/Publishing.md]: Follow the pre-publishing steps?

File: doc/Publishing.md [hunk 3]
```diff
@@ -2,10 +2,19 @@
...
 - login to the npm registry
 - publish packages
 - tag the published version
 
+## Pre-Publishing Steps
+Before publishing it's important to make sure that theia-apps builds against 'next'. Else we will have problems with "latest" after publishing
```
Comment (kittaakos) [inline doc/Publishing.md]: Please make sure we consistently use 'next' and "next". It is up to you which one you use, just pick one.

Comment (akosyakov) [reply doc/Publishing.md]: or `next` :)

**Comment by marcdumais-work** [comment]
Thanks guys - I have made the suggested corrections.

**Comment by kittaakos** [comment]
Ahh just missed it. Still has errors, did you read it through. This contains typos and other grammar mistakes. I also find it unprofessional when product and brand names are not correctly used; travis vs Travis. Please change. I can help, just let me know.

**Comment by kittaakos** [comment]
Or just please create a follow-up (if we do not have one yet) where we go through the documentation.

**Comment by marcdumais-work** [comment]
I see such a follow-up is already created: https://github.com/theia-ide/theia/pull/1167 .

**Pull request merged**